docs/assets/diagrams/linking/3-unlinking.puml

```diff
@@ -0,0 +1,58 @@
+@startuml Unlinking
+
+title: Unlinking
+hide footbox
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  participant "Firebase" as Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+'start flow
+Alice -> App++: I want to unlink this bank account
+
+App -> Server ++: ""**DELETE /consents/123**""
+App <-- Server: ""202 Accepted""
+deactivate App
+
+Server -> Adapter ++: ""**DELETE /consents/123**""
+Adapter <-- Server: ""202 Accepted""
+deactivate Server
+
+Adapter -> Server++: ""**PATCH /consents/123** \n{\n status: string,\n revokedAt: string (of timestamp)\n}""
+Adapter <-- Server: ""200 OK""
+deactivate Adapter
+
+rnote left of Server #Light
+  ""firebase.firestore()""
+  ""  .collection("consents")""
+  ""  .where("consentId", "==", "123")""
+  ""  .set({status: "REVOKED", revokedAt: "timestamp"})""
+end rnote
+
+Server -> Firebase: The account has been unlinked.
```

Comment:
```suggestion
Server -> Firebase ++: Update consent document
```

Reply:
again why is this better?

Reply:
I think because it's exactly what the server is doing (?)
And also because the app is the one who will say something like "The account has been unlinked" to the user according to all business logic in the mobile app